src/ArcCommander/ArgumentProcessing.fs

```diff
@@ -10,10 +10,37 @@ open System.Diagnostics
 open ISA.DataModel
 
 open Argu
-open Argu.ArguAttributes
 
 module ArgumentProcessing = 
 
+    type Argument =
+        | Field of string
+        | Flag of bool
```

Comment:
Why the bool type? Can't we just use the Union case `Flag` ? Or is this necessary downstream?

Reply:
I did it this way to allow for including this flag in the editor prompt (asking for additional input params) like this:

```
#Example Tooltip for field
FieldName:
#Example Tooltip for flag
FlagName: false
```
Putting it like this

    type Argument =
        | Field of string
        | Flag

would necessitate dropping the value from the argumentlist altogether to signal it not being set. This would make the editor prompt a bit more complicated to look like above.
But, do you think it even makes sense to include a flag in the prompt like above?

Reply:
If you offer the editor settings it is necessary to mimic the command line tool behaviour, so setting a boolean there is not correct i think. Can't you just use a tooltip without the prefilled line below?

```
#Example Tooltip for field
FieldName:
#Example Tooltip for flag1
#Example Tooltip for flag1
```